src/utils.py

```diff
@@ -19,9 +20,9 @@
     ('4k', [r'4k', r'2160[p]', r'uhd', r'4k', r'hd4k']),
     ('2k', [r'1440[p]', r'2k']),
     ('1080p', [r'1080[ip]', r'1920x1080', r'hd1080p?', r'fullhd', r'fhd', r'blu\W*ray', r'bd\W*remux']),
-    ('720p', [r'720[p]', r'1280x720', r'hd720p?', r'hd\-?rip', r'b[rd]rip']),
-    ('480p', [r'480[p]', r'xvid', r'dvd', r'dvdrip', r'hdtv', r'web\-(dl)?rip', r'iptv', r'sat\-?rip',
-              r'tv\-?rip']),
+    ('720p', [r'720[p]', r'1280x720', r'hd720p?', r'hd\-?rip', r'b[rd]rip', r'xvid', r'dvd', r'dvdrip', r'hdtv',
+              r'web\-(dl)?rip', r'iptv']),
+    ('480p', [r'480[p]', r'sat\-?rip', r'tv\-?rip']),
```

Comment:
Are there any examples for these? dvd is usually 480p or less. Not 720p. I see hdtv as the only one correctly moved option (in that, hdtv should be 720p). DVD/rip/xvid doesn't feel like it should be there.

Reply:
https://www.reddit.com/r/datacurator/comments/uy4uyy/is_webdl_dvdrip_hdrip_the_same_as_720p_or_1080p/

DVD can easily store 720p and 1080p movie. Moreover I would say most of the DVDs are 1080p. 

Xvid is a compression method, it can be 480p or 720p or 1080p. Am I wrong?
So I want this filter to be positive.

Reply:
I don't think it's about what one can do, it's about what the release scene does generally. I've never seen a xvid/dvd/dvdrip contain 720p content. I'd rather have this reverted.